pages/linux/nft.md

```diff
@@ -0,0 +1,36 @@
+# nftables
+
+> Program that allows configuration of tables, chains and rules provided by the Linux kernel firewall.
+> Nftables replaces iptables.
+
+- View current configuration:
+
+`sudo nft list ruleset`
+
+- Add a new table:
+
+`sudo nft add table {{family}} {{table}}`
+
+- Add a new chain:
+
+`sudo nft add chain {{family}} {{table}} {{chain}} \{ type {{type}} hook {{hook}} priority {{priority}} \; \}`
+
+- Add a new rule:
+
+`sudo nft {{[add | insert]}} rule {{family}} {{table}} {{chain}} {{[position]}} {{statement}}`
+
+- Example rule:
+
+`sudo nft add rule inet filter input tcp dport \{ telnet, ssh, http, https \} accept`
```

Comment:
This can be merged with the "Add a new rule" example. It is how we usually do it. You can expand the description a bit more mentioning the family, table and other terms. But we need to be careful to be make it too large. And use this concrete example there.

You can see the `ufw.md` page as a reference.